orcid-commons/dynamodb-local-metadata.json

```diff
@@ -0,0 +1 @@
+{"installationId":"d308586f-d8b2-493c-8135-21b55bdeb4e5","telemetryEnabled":"true"}
```

Comment:
These files should be excluded in gitignore, I guess

Reply:
Yes, good catch.